build.gradle

```diff
@@ -54,7 +54,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.4'
+        classpath 'com.android.tools.build:gradle:4.2.1'
```

Comment:
Is it safe for the whole app to update gradle tools? is whole app then compatible?

Reply:
Did not notice any problems when building the project, also integrating the new pickpic snapshot was working in lovoo :)